src/main.rs

```diff
@@ -41,11 +41,27 @@ fn parse_speed(arg: &str) -> Result<u8, String> {
     Ok(s)
 }
 
+fn parse_color_space(s: &str) -> Result<ColorSpace, String> {
+    match s.to_lowercase().as_str() {
+        "srgb" => Ok(ColorSpace::Srgb),
+        "displayp3" => Ok(ColorSpace::DisplayP3),
+        "rec2020Pq" => Ok(ColorSpace::Rec2020Pq),
+        _ => Err(format!("invalid color-space: {}", s)),
+    }
+}
+
 fn run() -> Result<(), BoxError> {
     let args = Command::new("cavif-rs")
         .version(clap::crate_version!())
         .author("Kornel Lesiński <kornel@imageoptim.com>")
         .about("Convert JPEG/PNG images to AVIF image format (based on AV1/rav1e)")
+        .arg(Arg::new("color-space")
+            .short('C')
+            .long("color-space")
+            .value_name("sRGB/displayP3/rec2020Pq")
+            .value_parser(parse_color_space)
+            .required(true)
+            .help("The color space passed values are in [possible values: sRGB, displayP3, rec2020Pq] "))
```

Comment:
```suggestion
            .help("The color space passed values are in [possible values: sRGB, displayP3, rec2020Pq]"))
```

Reply:
Note that your `parse_color_space()` ignores casing, perhaps this should be written without any, too.

(Though personally I favour sticking to case-sensitive cmdline arguments...)